lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -2,8 +2,18 @@
 
 const BbPromise = require('bluebird');
 const async = require('async');
+const path = require('path');
 
 module.exports = {
+  writeCreateTemplateToDisk() {
+    const cfTemplateFilePath = path.join(this.serverless.config.servicePath,
+      '.serverless', `cf-template-create-${(new Date).getTime().toString()}.json`);
```

Comment:
@pmuens @flomotlik I added a date to the name to keep it inline with the zip package.

Do you think a date is necessary here? It makes it hard to test and if the `.serverless` folder is being cleaned up it's probably unnecessary.